caproto/server/common.py

```diff
@@ -491,10 +491,8 @@ def get_db_entry():
             await self._cull_subscriptions(
                 db_entry,
                 lambda sub: sub.subscriptionid == command.subscriptionid)
-            try:
-                data_count = len(db_entry.value)
-            except (TypeError, ValueError):
-                data_count = 0  # or maybe 1?
+            # TODO: shouldn't this actually be the data_count requested?
```

Comment:
Yes. I think the catvs PR fixed some aspect of this but evidently not every aspect.